internal/runners/languages/languages.go

```diff
@@ -51,8 +68,41 @@ func (l *Languages) Run() error {
 		return locale.WrapError(err, "err_fetching_languages", "Cannot obtain languages")
 	}
 
+	// Fetch resolved artifacts list for showing full version numbers.
+	// Note: any errors here are not fatal, and only some of them should be reported to rollbar.
```

Comment:
I'm not sure I agree with this. Sure we could just show the languages without resolved version information, but we changed our stance on implicit errors on this quite a while ago to be explicit. ie. error out so we can fix the issue once and for all.

I won't mark each logged error in the review, but assume most if not all of them should be changed to error returns for this review iteration.